docs/CONFIG.md

```diff
@@ -212,8 +212,8 @@ of tokens before they browse.
 
 ### config\["get-more-passes-url"\]
 
-A valid URL string specifying an url where users can get more signed tokens by solving a 
-challange/CAPTCHA. This URL is pulled into the pop-up `Get more passes` menu item. 
+A valid URL string specifying an url where users can get more signed tokens by solving a
```

Comment:
While at it, maybe `an url` -> `a URL`?

Reply:
For sure! Just did it and squash the commits ;)